lambdas/evaluate-gpg45-scores/src/main/java/uk/gov/di/ipv/core/evaluategpg45scores/EvaluateGpg45ScoresHandler.java

```diff
@@ -60,15 +61,13 @@
 import static uk.gov.di.ipv.core.library.helpers.LogHelper.LogField.LOG_ERROR_JOURNEY_RESPONSE;
 import static uk.gov.di.ipv.core.library.helpers.LogHelper.LogField.LOG_MESSAGE_DESCRIPTION;
 import static uk.gov.di.ipv.core.library.helpers.LogHelper.LogField.LOG_MITIGATION_JOURNEY_RESPONSE;
-import static uk.gov.di.ipv.core.library.statemachine.BaseJourneyLambda.JOURNEY_ERROR_PATH;
 
 /** Evaluate the gathered credentials against a desired GPG45 profile. */
-public class EvaluateGpg45ScoresHandler implements RequestHandler<JourneyRequest, JourneyResponse> {
-
+public class EvaluateGpg45ScoresHandler extends JourneyRequestLambda {
     private static final List<Gpg45Profile> ACCEPTED_PROFILES =
             List.of(Gpg45Profile.M1A, Gpg45Profile.M1B);
-    private static final JourneyResponse JOURNEY_END = new JourneyResponse("/journey/end");
-    private static final JourneyResponse JOURNEY_NEXT = new JourneyResponse("/journey/next");
+    private static final JourneyResponse JOURNEY_END = new JourneyResponse(JOURNEY_ERROR_PATH);
```

Comment:
Why is `JOURNEY_END` changing from `/journey/end` to `/journey/error`

Reply:
I think this chest infection is addling my brain. Fixing that now.